2025/109-roy-jupyter.md

```diff
@@ -27,9 +27,64 @@ JupyterLab extensions can customize or enhance any part of JupyterLab. They can
 
 JupyterLab began in 2015 thanks to a $6 million grant from the Leona M. and Harry B. Helmsley Charitable Trust, the Gordon and Betty Moore Foundation and the Alfred P. Sloan Foundation. The first stable release was in 2018. Real time collaboration was added in 2021.
 
-```
 ## Timestamps
-00:00 Help us add timestamps
+```
+00:00 About Data Umbrella
+03:15 About the Presentation and Presenter
+04:00 CrossCompute
+04:25 CrossCompute Product History
+05:33 Sandbox Tools for Testing JupyterLab Extensions and Extension Development
+06:12 Overview
+06:50 Brief History of the Jupyter Project
+07:40 Jupyter Notebook vs JupyterLab
+08:41 Cool Features of JupyterLab that You Might Not Know
+10:25 JupyterLab Basics: Installation, Essential Keyboard Shortcuts
+12:20 JupyterLab Basics: Notebook Pre-Commit Hooks for Clearing Outputs
+13:39 JupyterLab Basics: Notebook Pre-Commit Hooks for Linting using nbqa
+14:28 JupyterLab Basics: Notebook Diffs to Compare Changes using nbdime
+15:12 JupyterLab Basics: Activating the JupyterLab Debugger kernel using xeus-python
+16:22 JupyterLab Basics: Try Untrusted JupyterLab Extensions in a Sandbox Environment
+17:05 JupyterLab Basics: Vim Key Bindings for JupyterLab with jupyterlab-vim
+17:33 JupyterLab Basics: Interactive Matplotlib Plots for JupyterLab with ipympl
```

Comment:
```suggestion
12:20 -- Notebook Pre-Commit Hooks for Clearing Outputs
13:39 -- Notebook Pre-Commit Hooks for Linting using nbqa
14:28 -- Notebook Diffs to Compare Changes using nbdime
15:12 -- Activating the JupyterLab Debugger kernel using xeus-python
16:22 -- Try Untrusted JupyterLab Extensions in a Sandbox Environment
17:05 -- Vim Key Bindings for JupyterLab with jupyterlab-vim
17:33 -- Interactive Matplotlib Plots for JupyterLab with ipympl
```

Reply:
Thanks for the suggestions -- give me a few minutes -- I will try to truncate the timestamps